testsuite/forge/src/backend/k8s/mod.rs

```diff
@@ -21,6 +25,56 @@ impl K8sFactory {
             treasury_compliance_key,
         }
     }
```

Comment:
I think we can get rid of this constructor once forge can load keys from vault directly. Also please clean & tidy up all other places

Reply:
done! lmk your comments